tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -284,6 +284,7 @@ def test_delete_restore_experiment_with_runs(self):
         self.assertEqual(len(restored_run_list), 2)
         for restored_run in restored_run_list:
             self.assertEqual(restored_run.lifecycle_stage, entities.LifecycleStage.ACTIVE)
+            self.assertTrue(restored_run.delete_time is None)
```

Comment:
```suggestion
            self.assertTrue(restored_run.deleted_time is None)
```

Reply:
This didn't work as `deleted_time` is no longer an attribute of `RunInfo`. I have changed it to query for `SqlRun` for the `deleted_time` attribute instead